act-rules-format.bs

```diff
@@ -110,42 +110,90 @@ An ACT Rule MUST have a description that is in plain language, and provides a br
 
+Each [=accessibility requirement=] in the mapping MUST include the following:
 
-Accessibility Requirements List {#accessibility-requirements-list}
-------------------------------------------------------------------
+1. the name, title or summary of the accessibility requirement, and
+2. the name of the standard or policy that includes the accessibility, and
+3. the conformance level associated with the accessibility requirement, if one exists.
 
-An ACT Rule SHOULD list an identifier and reference for each accessibility requirement that it maps to.
+For each accessibility requirement in the mapping, an ACT Rule MUST indicate what the outcome means for satisfying that accessibility requirement. When the outcome of a rule is `failed`, the accessibility requirements are *<dfn>not satisfied</dfn>*. When the outcome of a rule is `passed` or `inapplicable`, the accessibility requirements could be *<dfn>satisfied</dfn>*, or *<dfn>further testing could be necessary</dfn>*. Rules that can be used to determine if an accessibility requirement is *satisfied* are called <dfn>satisfying tests</dfn>.
 
-<div class=example>
-  <p><strong>Example</strong>: The outcome of this rule affects conformance to the following WCAG 2.1 success criteria:</p>
-  <ul>
-    <li>[1.3.1 Info and Relationships](https://www.w3.org/TR/WCAG21/#info-and-relationships)</li>
-    <li>[4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value)</li>
-  </ul>
+<div class=note>
+**Note**: In WCAG 2.1, success criteria do not evaluate to `passed`, `failed` or `inapplicable`. Rather they can be *satisfied* (or not). (See the [WCAG 2.1 definition: satisfies a success criterion](https://www.w3.org/TR/WCAG21/#dfn-satisfies).) If a success criterion is *not satisfied* a web page can only conform if there is a conforming alternative version, as described in [WCAG 2.1 Conformance Requirement 1](https://www.w3.org/TR/WCAG21/#cc1).
 </div>
 
-One reason why an [=atomic rule=] might not list accessibility requirements is because it is part of a [=composite rule=], and only a `failed` [=outcome=] in the composite rule would mean not satisfying the accessibility requirement. Another reason might be that an ACT Rule is designed to evaluate a best practice that is not part of any accessibility standard.
-
-Accessibility Requirements Mapping {#accessibility-requirements-mapping}
-------------------------------------------------------------------------
+<div class=example>
+  <p><strong>Example:</strong> Accessibility Requirements Mapping for a rule that tests if an image button has an accessible name:
+  <blockquote><ul>
+    <li><a href="https://www.w3.org/TR/WCAG21/#non-text-content">
+        Success Criterion 1.1.1: Non-text content
+      </a><ul>
+        <li><strong>Required for conformance</strong> to WCAG 2.0 and WCAG 2.1 level A</li>
```

Comment:
Is this "required/not required for conformance" even mentioned in the specification outside of this example?

Reply:
... it is apparently mentioned below this example, which seems a bit odd...